libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/dh-market-participant-edit-actor.component.ts

```diff
@@ -45,7 +46,9 @@ import {
   GridAreaDto,
   EicFunction,
   MarketRoleDto,
+  ActorContactDto,
 } from '@energinet-datahub/dh/shared/domain';
+import { DhMarketParticipantActorContactDataComponentScam } from './contact-data/dh-market-participant-actor-contact-data.component';
```

Comment:
Nitpick: order the imports 
1. Third party deps 
2. Internal deps from other libraries 
3. Relative deps